app/lib/formatters.ts

```
@@ -30,3 +30,12 @@ export function getWeekdayFromDate(
 
   return formatDate(date).weekdayLong || "";
 }
+
+export function slugify(input: string): string {
+  return input
+    .normalize("NFKD")             // strip accents
+    .replace(/[\u0300-\u036f]/g, "") // remove diacritics
```

Biome (1.9.4)

[error] A character class cannot match a character and a combining character.

A character and a combining character forms a new character. Replace the character class with an alternation.

(lint/suspicious/noMisleadingCharacterClass)